flax/nn/linear.py

```diff
@@ -181,8 +181,8 @@ def apply(self,
             kernel_size,
             strides=None,
             padding='SAME',
-            lhs_dilation=None,
-            rhs_dilation=None,
+            input_dilation=None,
```

Comment:
I like this but it is a breaking change. We should probably send an email to flax-announce@ to make people aware

Reply:
(And add the inaugural entry in CHANGELOG :) )